builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -344,6 +374,14 @@ func resourceAwsElasticacheClusterUpdate(d *schema.ResourceData, meta interface{
 		requestUpdate = true
 	}
 
+	if d.HasChange("snapshot_window") {
```

Comment:
`snapshot_window` and `snapshot_retention_limit` are not toggling the `requestUpdate` bit, so they're never actually being sent in any `update` call.